# import pymysql

# try:
#     conn = pymysql.connect(
#         host='localhost',
#         port=3306,
#         user='root',
#         password='kaliotman',
#         database='dcrm',
#     )
#     print('Connection successful!')
#     conn.close()
# except Exception as e:
#     print('Connection failed:', str(e))
